### PR TITLE
Adding test for for null endpoint auth parameter

### PR DESCRIPTION
**Pull request by omeshp**

### CODE REVIEW

File: src/Test/L0/Worker/WorkerL0.cs [hunk 1]
```diff
@@ -27,6 +27,10 @@ private JobRequestMessage CreateJobRequestMessage(string jobName)
             TaskOrchestrationPlanReference plan = new TaskOrchestrationPlanReference();
             TimelineReference timeline = null;
             JobEnvironment environment = new JobEnvironment();
+            var serviceEndpoint = new ServiceEndpoint();
```
Comment (GitHubSriramB) [inline src/Test/L0/Worker/WorkerL0.cs]: can you validate that the tests fail without the fix?
